README.md

```diff
@@ -40,8 +40,6 @@ If you would like to build from source:
 git clone https://github.com/helix-editor/helix
 cd helix
 cargo install --path helix-term
-hx --grammar fetch
-hx --grammar build
 ```
 
 This will install the `hx` binary to `$HOME/.cargo/bin` and build tree-sitter grammars.
```

Comment:
```suggestion
This will install the `hx` binary to `$HOME/.cargo/bin` and build tree-sitter grammars.
Tree-sitter grammars may be manually fetched and built with `hx --grammar fetch` and
`hx --grammar build`.
```

Reply:
Thank you, I reworded it slightly if you don't mind 🙂

Reply:
The only nit I have is that it's not really an alternative: grammars are automatically fetched and built in the `cargo install` step and then `hx --grammar fetch` and `hx --grammar build` are meant to be used after installation if you want to customize your `languages.toml` for example

Reply:
Ah, yeah, makes sense! Seems like important info, maybe we could mention it, e.g. "If you want to customize your `languages.toml` config, tree-sitter grammars may be manually fetched and built with `hx --grammar fetch` and `hx --grammar build`."

Reply:
I think that sounds good 👍